ppx/test/issue-429.t/run.t

```diff
@@ -44,7 +44,7 @@ Let's test hovering over parts of the component
         "line": 36,
         "col": 1
       },
-      "type": "(~greeting: string) => React.element",
+      "type": "(~greeting: string) => element",
```

Comment:
These changes are strange, I assume is the new merlin version used? I can't see anything in reason-react itself that changed, or in melange.

Reply:
I don't understand why these changed, but they're both correct and consistent across opam / nix calls, so I didn't worry too much.